eth/backend.go

```diff
@@ -549,8 +545,6 @@ func (s *Ethereum) Start() error {
 	if err != nil {
 		return err
 	}
-	// periodically flush databases
-	go s.syncDatabases()
```

Comment:
It has been a no-op for LevelDB databases since https://github.com/ethereum/go-ethereum/commit/020006a8ede0463346ad3d4ae318d299eee63fb.